protos/class.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+import "protos/user.proto";
+import "protos/chat.proto";
+import "protos/error.proto";
+import "protos/quiz.proto";
+
+service PresenterClassService {
+    rpc CreateClass (ClassCreationRequest) returns (Error);
+    rpc UpdateClass (ClassUpdateRequest) returns (Error);
+    rpc DeleteClass (ClassDeleteRequest) returns (Error);
+
+    rpc StartClass (ClassStartRequest) returns (Error);
+    rpc DelegateQuizQuestion (ClassQuizQuestion) returns (Error);
+    rpc EndClass (ClassEndRequest) returns (Error);
+}
+
+service StudentClassService {
+    rpc JoinClass(JoinClassRequest) returns (stream ClassEvent);
+    rpc SendMessageToPresenter(StudentMessage) returns (Error);
+}
+
+// TODO
+message ClassCreationRequest {
+}
+message ClassUpdateRequest {
+}
+message ClassDeleteRequest {
+}
+message ClassStartRequest {
+}
+message ClassEndRequest {
+}
+message ClassEvent{
+    // example
+    repeated ClassQuizQuestion classQuizQuestion = 1;
+}
+message StudentMessage{
+}
+
+message JoinClassRequest {
+}
```

Comment:
taki kod nie powinien znalezc sie na masterze, jesli chcesz zostawic uzupełnienie mi to zrob PR do mojego brancha feature/3 zamiast do mastera